backend/models/School.js

```diff
@@ -0,0 +1,100 @@
+const Class = require('./Classes');
+const Student = require('./Student')
+// const SchoolRouter = require("./School.js/index.js.js")
+
+// const express = require("express")
+// const router = express.Router();
+
+class School {
+  constructor() {
+    this.classes = {
+      // className: Class Object
+      //   physics: {} 
+    }
+  }
+
+  /**
+   * Add class to classes
+   * 
+   * @param {string} name - Name of the class
+   * @param {string} teacher - Name of instructor 
+   * @return {Class} Class object
+   */
+
+  addClass(name, teacher) {
```

Comment:
You need to return the `newClass` in this function so you can later include it in your response